### DISCUSSION TITLE
챕터 3. 웹사이트 성능을 개선하는 기본적인 방법

### DISCUSSION THREAD

**Discussion by yujo11** [Frontend-Divers/performance-optimization#15] [함께 얘기하고 싶은 내용]
챕터 3. 웹사이트 성능을 개선하는 기본적인 방법

**Comment by Tanney-102** [comment]
- s3에 gzip 방식으로 압축된 스크립트를 업로드해도 정작 브라우저에서는 압축되지 않은 스크립트를 요청했던 경험이 있음.
- HTTP 헤더를 적절히 설정하지 않았기 때문인지 궁금함
- 이번 주안에 실험해보겠습니다.

**Comment by Tanney-102** [comment]
- 캐시에 대해 서로 알고있는 내용을 나눠봤으면 좋겠습니다.
- 하드웨어 측면에서의 캐시부터 인터넷 캐시까지!
- 과거 cloudfront에서 각 엔드 포인트 외 캐시 서버 계층을 하나 더 두어 캐시 적중률을 높인다는 말을 듣고 '그게 뭔 상관이지' 싶었는데 ICP와 관련 있는 것인지 궁금하네요

**Reply by Tanney-102** [reply]
지금 advanced로 캐시에 대한 전반적인 내용을 적어보고 있는데 일정상 오늘안에 적기는 힘들 것 같습니다ㅠㅠㅠ 이것도 이번주 안에 작성해서 올리도록 할게요!

**Reply by al-bur** [reply]
> 지금 advanced로 캐시에 대한 전반적인 내용을 적어보고 있는데 일정상 오늘안에 적기는 힘들 것 같습니다ㅠㅠㅠ 이것도 이번주 안에 작성해서 올리도록 할게요!

네네!!
